### ISSUE TITLE
Dial background for text dials different than for thumbnail dials (defaults) ?

### ISSUE THREAD

**Issue by tockschmock** [fastaddons/GroupSpeedDial#191]
Text Dials show another background than thumbnail dials created from the public library:

Is there a way to keep the text dial background equal to the thumbnail dial background?

![Screenshot 2022-11-08 18 46 48](https://user-images.githubusercontent.com/68165681/200638985-d1bc4d5b-f3d1-4ee7-91d2-04b1e301a0d5.png)

**Comment by fastaddons** [comment]
Try "Edit dial" window / Visuals / Background color and set it to transparent.

**Comment by cipricus** [comment]
That was already answered [HERE](https://github.com/fastaddons/GroupSpeedDial/issues/178#issuecomment-1305468910) - if you care to read my carefully written comments before posting a river of questions ;)

**Comment by tockschmock** [comment]
please forgive me, if I do things not in an programers professional way here

i will stop my activities, if they are not useful

i can try to make sure, as best as I can, that questions don't duplicate, but cannot guarantee knowing all the around 200 post here

**Comment by cipricus** [comment]
> i can try to make sure, as best as I can, that questions don't duplicate, but cannot guarantee knowing all the around 200 post here

No worries. It's just you might not get the answer. But you might as well get it anyway. 

To look at previous issues search like this:

![image](https://user-images.githubusercontent.com/3447868/200824476-6d98b2dc-2629-4a38-9db1-d2ec0dab0546.png)

**Comment by tockschmock** [comment]
O.K.

**Comment by tockschmock** [comment]
> Try "Edit dial" window / Visuals / Background color and set it to transparent.

there is no such setting like transparent for the dial background color in "Visual"
this is how the default look of ordinary dials and text dials differs on my Windows

![image](https://user-images.githubusercontent.com/68165681/200948335-dfeeb88f-f8c6-4d68-8e87-57fe1fef4076.png)

**Comment by cipricus** [comment]
But you can set what color you want for it.

**Comment by cipricus** [comment]
> there is no such setting like "transparent" for the dial background color in "Visual"

You're right, i didn't read your post correctly. By "text dial background" I though you mean "the background of dial text/name" (but even for that I was wrong: the correct answer is to check the general option Dials>Layout>"Transparent dial name background (puts focus on thumbnail)")

**Comment by tockschmock** [comment]
Thanks.
I understand now that not only text dials, but also other dials (I tested the _recently-closed tabs_ dial and the _top sites_ dial) show the different background.
But my question remains:
Can I also make these dials transparent, i.e. showing the group background within the dials similar to the appearance of the dials with thumbnails?

**Comment by fastaddons** [comment]
Hello again,
There seems to be an issue with the "default" settings and transparent colors are not accessible in the popup.
But setting the color manually to "#00000000" will make it transparent. Or "#FFFFFF00" will also work.

**Comment by tockschmock** [comment]
Thanks - that works (requires a setting for each dial separately)
I think, the background setting (_#00000000_ = transparent) should be the standard for all new dials.
Currently this is not the case at least for the;
- text-
- recently-closed-
- top sites-

dials (not all tested).


I would be also nice, if more operations could be done on a selection of dials at once:
- delete thumbnail
- set thumbnail (same thumbnail might be good enough for a selection)
- set background
- visuals
- auto reload

Currently only _Cut_ and _Move to Group_ is offered for a multiple selection.
